v03_pipeline/lib/misc/io_test.py

```diff
@@ -17,3 +24,97 @@ def test_compute_hail_n_partitions(self) -> None:
         self.assertEqual(compute_hail_n_partitions(191310), 1)
         self.assertEqual(compute_hail_n_partitions(1913100000), 15)
+
+    def test_annotate_vets(self) -> None:
+        gatk_mt = hl.MatrixTable.from_parts(
+            rows={
+                'locus': [
+                    hl.Locus(
+                        contig='chr1',
+                        position=1,
+                        reference_genome='GRCh38',
+                    ),
+                ],
+                'filters': [
+                    hl.set(['PASS']),
+                ],
+            },
+            cols={'s': ['sample_1']},
+            entries={'HL': [[0.0]]},
+        ).key_rows_by('locus')
+        gatk_mt = annotate_vets(gatk_mt)
+        dragen_mt = hl.MatrixTable.from_parts(
+            rows={
+                'locus': [
+                    hl.Locus(
+                        contig='chr1',
+                        position=1,
+                        reference_genome='GRCh38',
+                    ),
+                    hl.Locus(
+                        contig='chr1',
+                        position=2,
+                        reference_genome='GRCh38',
+                    ),
+                    hl.Locus(
+                        contig='chr1',
+                        position=3,
+                        reference_genome='GRCh38',
+                    ),
+                    hl.Locus(
+                        contig='chr1',
+                        position=4,
+                        reference_genome='GRCh38',
+                    ),
+                    hl.Locus(
+                        contig='chr1',
+                        position=5,
+                        reference_genome='GRCh38',
+                    ),
+                    hl.Locus(
+                        contig='chr1',
+                        position=5,
+                        reference_genome='GRCh38',
+                    ),
+                ],
+                'alleles': [
+                    ['A', 'T'],
+                    ['A', 'T'],
+                    ['A', 'T'],
+                    ['AC', 'T'],
+                    ['AT', 'TC'],
+                    ['AG', 'TG'],
+                ],
+                'filters': [
+                    hl.set(['PASS']),
+                    hl.empty_set(hl.tstr),
+                    hl.missing(hl.tset(hl.tstr)),
+                    hl.set(['PASS']),
+                    hl.empty_set(hl.tstr),
+                    hl.set(['PASS']),
+                ],
+                'info': [
+                    hl.Struct(CALIBRATION_SENSITIVITY=['0.999']),
+                    hl.Struct(CALIBRATION_SENSITIVITY=['0.995']),
+                    hl.Struct(CALIBRATION_SENSITIVITY=['0.999']),
+                    hl.Struct(CALIBRATION_SENSITIVITY=['0.98']),
+                    hl.Struct(CALIBRATION_SENSITIVITY=['0.99']),
+                    hl.Struct(CALIBRATION_SENSITIVITY=['0.991']),
+                ],
```

Comment:
At the moment, VETS is only ran on variants that pass all other filters and are in high-quality genotypes, essentially. Variants with 'AC0' and 'ExcessHet' don't have VETS scores. This is by design, and the same behavior as VQSR. 

Would we want to include a test case to future-proof ourselves if this behavior changes, or is that a future Ben/Dan problem

Reply:
that might be outside our purview... 

would we actually want to add code that says `if AC0 is present don't check VETs`?  that's a line of code we can add a test for!